### ISSUE TITLE
Peer-Peer or WiFi Access point

### ISSUE THREAD

**Issue by sdeep211** [rahul-thakoor/air-pi-play#30]
I would like to use your image with a Pi Zero but connect to the Pi without using a router - Peer-Peer by using WiFi Access point
From [https://github.com/FD-/RPiPlay/issues/9](https://github.com/FD-/RPiPlay/issues/9)
I could set up the pi zero as a WiFi access point.
Is this possible with your image or should I build using rpiplay?

I am already using balena-rpiplay with a Pi4 and as a teacher I really like your image for dedicated mirroring with my ipad.

**Comment by rahul-thakoor** [comment]
I don't see any reason why it wouldn't work. As long as the pi and ipad are on the same network it should be good.

How are you deploying the image?
Are you using balenaOS?

Thanks

**Comment by sdeep211** [comment]
Yes I would like to use balenaOS and deploy using the balena open or cloud fleet though using the pi in this way means it would have no internet connection.
I guess this will then work by using the balenaOS hotspot option: https://www.balena.io/docs/reference/OS/network/2.x/#creating-a-hotspot

I will give it a try and post my results and close this issue once I have.

Thanks!

**Comment by sdeep211** [comment]
I did try to get this working with Balena OS and a Pi Zero W (1st Gen) but could not push the dockerfile using Balena-CLI

Resolving services with [raspberry-pi|rpi]
Could not parse FROM command on line 31
DockerfileParseError: Could not parse FROM command on line 31
    at Dockerfile.parse (/snapshot/versioned-source/node_modules/livepush/build/dockerfile.js:120:31)
 
that line being 'FROM --platform=linux/arm/v6 busybox:stable'

I also tried to deploy the Development version of Balena Cloud OS and built and pushed balena-rpiplay via the cloud but the rpiplay service would not start.

Next I tried the regular cloud version which I could not ssh into.
Then the cloud development version which I can ssh into, no logo but I can cast.

Now to try the hotspot.

**Comment by sdeep211** [comment]
I configured the hotspot using the NetworkManager example from here [https://www.balena.io/docs/reference/OS/network/#creating-a-hotspot](url)
This did not work with screen mirroring.
The ipad I was using showed "weak security" so I added ccmp to pairwise and group.
This got rid of the "weak security" on the ipad when connected to the hotspot.
Unfortunately this did not work and no screen mirroring.
I am going to use a wifi router with the pi zero instead.

I don't have a pi4 to try the hotspot on and cannot think of anything else to try, so closing this issue.
